src/resources/elements/primeDesignSystem/prange-slider/prange-slider.ts

```diff
@@ -62,10 +54,8 @@ export class PRangeSlider {
       this.alreadyUpdated = false;
       return;
     }
-    this.left = this.numberService.toString(this.percentageToAbsoluteValue(100 - this.value), {mantissa: 0});
-    this.right = this.numberService.toString(this.percentageToAbsoluteValue(this.value), {mantissa: 0});
-
-    this.updateTooltips();
+    this.left = this.percentageToAbsoluteValue(Number((1 - this.value).toFixed(2)));
```

Comment:
we are supposed to be showing all the decimals.  That is why we got rid of the tooltips.

Reply:
`.toFixed(2)` is not used to show the actual value shown in the UI. On this line `.toFixed` is used to limit the percentage value to 2 decimal places between 0.00 and 1.00. `Number((1 - this.value).toFixed(2)` was actually added because Javascript returns `0.45999999999999996` when doing `1 - 0.54` making the `percentageToAbsoluteValue` return an error because BigNumbers can't handle decimal places like that. So I had to use `.toFixed(2)` to transform it to `0.46`.

Reply:
Still, we should be showing all the decimals in the left and right input fields.  That is why we got rid of the tooltips.